docs/content/components/forms.mdx

```diff
@@ -172,25 +269,35 @@ in the context of a form.
   ]} />
 </Playground>
 
-### Best practices
-- Use different shadows.
+### When to use it
+
+Best used when there are 4 or more defined options to choose from (e.g. Months).If there are less than 4 options, 
+consider using the radio button component.
 
 ### Usage guidelines
-- Every card should be a logical combination of its contents.
 
-### Component variations
+- **Do** clearly label the select dropdown so that the merchant knows what options will be available
+- **Do** always provide a default option, typically the placeholder. The default option is not 
+selectable but should specify what options will be available.
+- **Do** list the options in a logical order. (e.g. Alphabetical, numerical, or in some order that allows the 
+merchant to find the option as easy as possible)
+- **Do not** use it to save space or tidy up a form.
+- **Do not** use it as a way to filter content
 
 ## Textarea
 
+A text area input is a type of input that has more than one line of text. Such
+types of input are used when the expected text is more than one line.
```

Comment:
This sounds like a circular definition to me 😁 

Also, `one line` is ambiguous for a length. Would it be better to say something like `5+ words`?

Reply:
Indeed! Since is the definition of the component I will change it to _"It is a type of input that allows multiple lines of text."_ Then I think that on the `When to Use` part it becomes clear the context! Makes sense?